src/molecules/product-list-item/ProductListItem.html

```diff
@@ -0,0 +1,128 @@
+<a-list-item tag="div" class="a-product-list-item">
+  <div class="a-product-list-item__image">
+    <!--@slot Custom product image (Scoped slot)-->
+    <slot
+      name="image"
+      :alt="name"
+      :image-url="imageUrl"
+    >
+      <a-image
+        :src="imageUrl"
+        :alt="name"
+      />
+    </slot>
+  </div>
+  <div class="a-product-list-item__info">
+    <div class="a-product-list-item__data">
+      <!--@slot Custom product name (Scoped slot)-->
+      <slot
+        name="name"
+        :product-name="name"
+        :product-url="url"
+      >
+        <router-link
+          :to="url"
+          class="link link--inverted a-product-list-item__name"
+        >
+          {{ name }}
+        </router-link>
+      </slot>
+      <!--@slot Custom product price (Scoped slot)-->
+      <slot
+        name="price"
+        :price="price"
+        :special-price="specialPrice"
+        :old-price="oldPrice"
+      >
+        <a-price
+          :price="price"
+          :special-price="specialPrice"
+          :old-price="oldPrice"
+          class="a-product-list-item__price-wrapper"
+          old-price-class="a-product-list-item__price-wrapper--old"
+          special-price-class="a-product-list-item__price-wrapper--special"
+        />
+      </slot>
+
+      <a-list
+        v-if="options"
+        tag="dl"
+        class="a-product-list-item__attributes"
+      >
+        <template v-for="item in options">
+          <!-- @slot Custom product item option (Scoped slot) -->
+          <slot
+            name="option"
+            :option="item"
+          >
+            <a-list-item tag="dt">
```

Comment:
It looks different on design 
![image](https://user-images.githubusercontent.com/5370251/59428662-27b5e900-8dde-11e9-9c67-f5d69e6bcf65.png)
Please think about adding styles for a descriptive list, maybe it makes sense to add them as list variant?

Reply:
You're right, it was related with some other changes in child components. Styles were added and used in this component, let me know what do you think :slightly_smiling_face:

Reply:
it looks a little strange if we have long labels 
![image](https://user-images.githubusercontent.com/5370251/60111283-f5a56f00-976d-11e9-8d88-6b3efb4fde43.png)
I'll try to change it, I think we have to use unfortunately `float` property. the same as here https://alpaca-components.now.sh/components/detail/list--description.html